Modules/_testcapi/util.h

```diff
@@ -23,3 +23,10 @@
         assert(!PyErr_Occurred());          \
         return PyLong_FromSsize_t(_ret);    \
     } while (0)
+
+/* Marker to check that pointer value was set. */
+#define UNINITIALIZED_PTR ((void *)"uninitialized")
+/* Marker to check that Py_ssize_t value was set. */
+#define UNINITIALIZED_SIZE ((Py_ssize_t)987654321)
+/* Marker to check that integer value was set. */
+#define UNINITIALIZED_INT (123456789)
```

Comment:
I suggest a more random number :-) For different reasons, 123456789 can be set by mistake ;-)

```suggestion
#define UNINITIALIZED_INT (63256717)
```